dynamixel_workbench:
  ros__parameters:
    usb_port:               /dev/ttyUSB0
    dxl_baud_rate:          1000000
    dxl_read_period:        0.010
    dxl_write_period:       0.010
    use_joint_states_topic: false
    use_moveit:             false
    dynamixel_info:         basic.yaml
    mobile_robot_config:
      separation_between_wheels: 0.500
      radius_of_wheel:           0.050
